### PR TITLE
can now receive the odd root plant from digging grass

### PR DESCRIPTION
**Pull request by ghost**
Didn't add garlic as that's a serious balance risk. Instead, just potato, carrot, and beet

:cl:
* rscadd: Digging grass can yield the odd seed or root plant

### CODE REVIEW

File: code/game/turfs/simulated/floor.dm [hunk 1]
```diff
@@ -596,6 +596,21 @@ turf/simulated/floor/update_icon()
 			new /obj/item/weapon/ore/glass(src)
 			new /obj/item/weapon/ore/glass(src) //Make some sand if you shovel grass
 			to_chat(user, "<span class='notice'>You shovel the grass.</span>")
+			if(prob(10))
+				switch(rand(1,6))
```
Comment (PJB3005) [inline code/game/turfs/simulated/floor.dm]: So, just use a pick over a set of typepath?

**Review by PJB3005 — CHANGES_REQUESTED** [review]

**Review by PJB3005 — APPROVED** [review]

**Comment by FudgePucker** [comment]
Speaking of roots. We should add onions. I know about 1000 different cultanary uses for onions, like two for beets.

**Comment by ghost** [comment]
>onion rings
>quiche
>french onion soup

yes

**Comment by Kurfursten** [comment]
We already have quiche but it's made with garlic

**Comment by SonixApache** [comment]
just like in my farming simulators

**Pull request merged**